### ISSUE TITLE
Add documentation for "git commit --date=now"

### ISSUE THREAD

**Issue by johnlinp** [gitgitgadget/git#302]
There are some special strings that we can pass to `--date` in `git commit`, e.g. `now`, `yesterday`, `noon`, `tea`...

Personally I think `--date=now` is the most useful because I will use it with `git commit --amend`. I guess we don't have to make an exhaustive list of special strings in the documentation, but I think we can at least mention `now`.

**Comment by jpninanjohn** [comment]
I can do this. Can someone please point me to where the docs are kept?

**Comment by dscho** [comment]
> Can someone please point me to where the docs are kept?

Gladly: https://github.com/git/git/tree/v2.23.0/Documentation/

You will probably see very quickly where the patch needs to go ;-)

**Comment by jpninanjohn** [comment]
https://github.com/gitgitgadget/git/pull/408

I have raised a pull request for it. I am not sure if I have fulfilled the commit message requirements correctly. Do tell if there is anything I have to correct.

**Comment by Rhythm-26** [comment]
@dscho I dont see PR to me merged, neither there is any response from the author for more than a year now, can i take this up?

**Comment by dscho** [comment]
@Rhythm-26 how about asking @jpninanjohn first?

**Comment by Rhythm-26** [comment]
Yes, lets do that. @jpninanjohn please let me know if you are working on it or if you'd prefer me to continue.

**Comment by dscho** [comment]
@Rhythm-26 now that more than two months of beautiful cricket sounds have filled the air, from my side you have the blessing to proceed without their consent.
